packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -661,7 +661,7 @@ export class SnapController extends BaseController<
 
   #rollbackSnapshots: Map<SnapId, RollbackSnapshot>;
 
-  #getAppKey: GetAppKey;
+  #getMnemonicPhrase: GetMnemonicPhrase;
```

Comment:
I wonder if we can access this in another way that doesn't require us passing a constructor arg.

We could move more of the state implementation to the RPC implementation, what do you think?

Reply:
I still feel like getting rid of the constructor arg is worth it. If you feel like it is out of scope, please open an issue for it.